website/blog/2023-01-04-ferretdb-beta-version-authentication.md

```diff
@@ -0,0 +1,117 @@
+Be sure to [check our roadmap for further details](https://github.com/orgs/FerretDB/projects/2).
+
+:::caution
+Please note that this particular change breaks backward compatibility.
+To dump and restore the data, you’ll need connection strings to connect to FerretDB and PostgreSQL.
+There are numerous ways to dump and restore your data.
+For example, you can follow the following steps:
+
+1. Backup FerretDB databases with `mongodump`.
+Set your FerretDB connection string in `—uri` and run:
+   `mongodump --uri="mongodb://localhost:27017"`
+   This command will create a directory with dumps for each FerretDB database on the given server.
+Later, we will use this dump to restore the database.
+Instead of `mongodump`, you can also use `mongoexport` for this.
+
+2. Backup PostgreSQL `ferretdb` database.
+If the migration goes well, we will not use this backup.
+But we recommend to do this step in case you need to rollback.
+Specify your host and port.
+`pg_dump -h localhost -p 5432 -U username ferretdb > ferretdb.sql`
+
+3. Stop FerretDB (This depends on your operating system and the way you run FerretDB)
+
+4. Connect to PostgreSQL and drop `ferredb` database as it's not needed anymore:
+   * `psql -h localhost -p 5432 -U username postgres`
+   * `DROP DATABASE ferretdb`
+
+5. Upgrade FerretDB and run FerretDB 0.8 (Please refer to our documentation where we describe how to update and start FerretDB)
+
+6. Restore database using `mongorestore --uri="mongodb://localhost:27017"`
+
+7. In case something doesn't work and you need to rollback to FerretDB 0.7.1:
+   * Stop ferretdb
+   * Delete ferretdb database (repeat step 4)
+   * Restore `ferretdb` PostgreSQL db from the dump we created on the step 2:  `psql -h localhost -p 5432 -U username ferretdb -f ferretdb.sql`
+   * Start ferretdb 0.7.1
+:::
+
+In this blog post, we'll be sharing detailed information about the FerretDB beta release (0.8.0).
+
+## What's new?
+
+Since the last release (0.7.2), we've made great strides in adding new features to FerretDB, especially the introduction of authentication for PostgreSQL.
+With authentication, users can now connect securely using passwords, ensuring that only authorized connections are established.
+You can implement an authenticated connection by specifying the connection string in this format: `postgres://username:password@postgres:5432/ferretdb`.
+See more details [in our documentation](https://docs.ferretdb.io/security/#authentication).
+
+But that's not all - in addition to the `$max` update operator, FerretDB beta now includes support for the `$min` update operator:
+
+```js
+db.collection.update( {}, { $min: { <field1>: <value1>, ... } } )
+```
+
+We've also added support for `ordered` inserts, which allows you to insert data in the exact order it comes in.
+
+```js
+db.collection.insert( { "field1": "value1" }, { ordered: true } )
```

Comment:
Those examples are not consistent: first one uses `<field1>: <value1>` "pseudocode", and the second one is a particular example. Should we make them written in the same way? 

Based on how they render, maybe multi-line could be more readable.

Reply:
Good spotting! I'll work on that now

Reply:
PTAL

Reply:
LGTM